fenix/app/src/main/java/org/mozilla/fenix/debugsettings/store/DrawerStatus.kt

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.debugsettings.store
+
+/**
+ * Possible values of the debug drawer's physical state.
+ */
+enum class DrawerStatus {
```

Comment:
nit: `DrawerState`?

Reply:
this would affect some of the variable names

Reply:
I've been hesitant to use `DrawerState` to avoid any naming overload confusion, since we have `DrawerState` from `ModalDrawer` and then `DebugDrawerState` from the redux.

Reply:
Ah okay, gotcha! Based on the desc for `DebugDrawerState` `UI state of the debug drawer feature.` it seems like this enum is a part of this?

Reply:
It is, but after the PR, it's now a value passed into `DebugOverlay` and then managed outside of it (so if this is uplifted to AC, it can be used with any framework). So in other words it's a dependency of the feature but then managed in Fenix via `DebugDrawerState`.

Reply:
👍 

In that case, I'm wondering if we could do with updating some of the naming around this, as I'd say this really is a 'state' over a 'status' and we're only using that because it's already used? Maybe `DebugDrawerState` should become something like `DebugDrawerUiState` or `DebugDrawerUiData`? 

`OnboardingPageUiData.kt` maybe a useful reference

Reply:
I hear you. I think I need to simmer on this, but I'll add a note to my todo to passively think about this in the next week or so. If you, or anyone else randomly reading this thread, come up with a way to keep our store/state naming convention while having an indicative name for the drawer open/close enum (that doesn't overlap with the Composable library naming) please send them my way!